### PR TITLE
Manage alignakdaemon backend endpoint. closes #53

### LINKED ISSUE #53: Feed alignakdaemon endpoint
Currently, `alignakdaemon` is not feed by the module and is empty. 

Only the model (see https://github.com/Alignak-monitoring-contrib/alignak-backend/issues/173) is present.

### PR DESCRIPTION
**Pull request by ddurieux**
Need tests and PR #54 merged to rebase on it

### CODE REVIEW

**Comment by mohierf** [comment]
Fix the PR title please :wink:

**Comment by ddurieux** [comment]
done :p

**Comment by mohierf** [comment]
Still WiP I suppose?

**Comment by ddurieux** [comment]
Yes still WiP because need finish the test (will be ok this evening)

File: alignak_module_backend/arbiter/module.py [hunk 1]
```diff
@@ -290,12 +297,16 @@ def get_realms(self):
...
         self.configraw['realms'] = {}
+        self.configraw['realms_name'] = {}
         all_realms = self.backend.get_all('realm', {'embedded': json.dumps({'_children': 1})})
         logger.info("Got %d realms",
                     len(all_realms['_items']))
         for realm in all_realms['_items']:
             logger.info("- %s", realm['name'])
             self.configraw['realms'][realm['_id']] = realm['name']
+            self.configraw['realms_name'][realm['name']] = realm['_id']
+            if realm['_level'] < self.highlevelrealm['level']:
```
Comment (mohierf) [inline alignak_module_backend/arbiter/module.py]: Why doing this? Please comment to explain...

Comment (ddurieux) [inline alignak_module_backend/arbiter/module.py]: Because arbiter not have realm information, so I get the most higher level realm

Comment (mohierf) [inline alignak_module_backend/arbiter/module.py]: Comment in the source code please :wink:

**Pull request merged**